### ISSUE TITLE
Autocomplete is not Accessable (Tested on NVDA)

### ISSUE THREAD

**Issue by zamamohammed** [angular/material#1473]
I have tested Autocomplete with NVDA and it reads "blank" instead of the list items when you use arrows to scroll through the list

**Comment by marcysutton** [comment]
@zamamohammed we worked on a fix and just pushed it to master in a787e2262fd805e1edd5fc5fd412a332ef673657. Can you try again?

**Comment by zamamohammed** [comment]
@marcysutton I am still hearing the 'blank'

Please listen to the recording:
https://www.dropbox.com/s/q9ezng1yocskjqg/Autocomplete.swf?dl=0

**Comment by marcysutton** [comment]
I see what you're saying. It's really slow to announce "There is one match available" or the selection on arrow down in Firefox/NVDA despite having `aria-live="assertive"` on the text to be announced. Sometimes it doesn't read at all. It's great in VoiceOver, but not so good in JAWS and NVDA (which have higher usage). I'll see if I can make it better.

**Comment by zamamohammed** [comment]
Most of the custom dropdown plugins in the Community (Chosen, Select2 etc) also has the same issue with it, the only implementation I liked is of Jquery UI (http://jqueryui.com/selectmenu/)

**Comment by marcysutton** [comment]
Good to know. I think we can get it working by improving our implementation. The inspiration for our autocomplete was actually Twitter's Typeahead/draft tweet functionality.

**Comment by robertmesserle** [comment]
@marcysutton I believe you were looking into this, assigned to you.

**Comment by zamamohammed** [comment]
@marcysutton when can I test this on material.angular.com ?

**Comment by marcysutton** [comment]
@zamamohammed it is up on the docs site now! Just not out in a release yet.
